### PR TITLE
Update test build for testing 7.1 beta conda recipe for linux and osx

### PR DESCRIPTION
**Pull request by jchodera**
@peastman @mpharrigan: I've been combining all of the fixes suggested to build the openmm 7.1 beta, but have run into a bizarre glibc segfault when building the python wrappers. Backtrace attached.

[backtrace.txt](https://github.com/omnia-md/conda-dev-recipes/files/662810/backtrace.txt)

To replicate, first start docker:
```bash
docker run -i -t jchodera/omnia-build-box:cuda80-amd30-clang38 /bin/bash
```
Then build inside the docker image:
```bash
# First, the boilerplate from devtools/docker-build.sh to mimic what conda-recipes does
source /hbb_exe/activate
unset PYTHONPATH
set -x
curl -s -O https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
bash Miniconda3-latest-Linux-x86_64.sh -b -p /anaconda
PATH=/opt/rh/devtoolset-2/root/usr/bin:/opt/rh/autotools-latest/root/usr/bin:/anaconda/bin:$PATH
conda config --add channels omnia
conda install -yq conda-build==2.0.1 jinja2 anaconda-client

# Clone conda-dev-recipes
git clone https://github.com/omnia-md/conda-dev-recipes.git
cd conda-dev-recipes; git checkout openmm-7.1-beta; git pull origin openmm-7.1-beta ; cd /

# Build conda package
wget -O conda-build-all --no-check-certificate --quiet https://raw.githubusercontent.com/omnia-md/conda-recipes/master/conda-build-all
chmod u+x conda-build-all
./conda-build-all -vvv --dev --force -- /conda-dev-recipes/openmm || true

# Upload package to anaconda cloud
#anaconda upload --force --label beta --channel omnia --user omnia /anaconda/conda-bld/linux-64/openmm-7.1.0-py* || true
```

### CODE REVIEW

**Comment by jchodera** [comment]
I just realized that I'm probably replacing the anaconda python by updating `$PATH` in the `build.sh` script. Trying to fix that.

**Comment by jchodera** [comment]
I've fixed the Python issue, but we get a different error on `travis` than I am seeing locally:
https://travis-ci.org/omnia-md/conda-dev-recipes/jobs/185365498#L1055-L1111
```
-- The C compiler identification is Clang 3.8.1
-- The CXX compiler identification is Clang 3.8.1
-- Check for working C compiler: /opt/clang/bin/clang
-- Check for working C compiler: /opt/clang/bin/clang -- broken
CMake Error at /anaconda/conda-bld/openmm_1482212188316/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/share/cmake-3.6/Modules/CMakeTestCCompiler.cmake:61 (message):
  The C compiler "/opt/clang/bin/clang" is not able to compile a simple test
  program.
  It fails with the following output:
   Change Dir: /anaconda/conda-bld/openmm_1482212188316/work/build/CMakeFiles/CMakeTmp
  
  Run Build Command:"/usr/bin/gmake" "cmTC_5ebcb/fast"
  /usr/bin/gmake -f CMakeFiles/cmTC_5ebcb.dir/build.make
  CMakeFiles/cmTC_5ebcb.dir/build
  gmake[1]: Entering directory
  `/anaconda/conda-bld/openmm_1482212188316/work/build/CMakeFiles/CMakeTmp'
  Building C object CMakeFiles/cmTC_5ebcb.dir/testCCompiler.c.o
  /opt/clang/bin/clang -g -O3 -I/hbb_shlib/include -o
  CMakeFiles/cmTC_5ebcb.dir/testCCompiler.c.o -c
  /anaconda/conda-bld/openmm_1482212188316/work/build/CMakeFiles/CMakeTmp/testCCompiler.c
  Linking C executable cmTC_5ebcb
  
  /anaconda/conda-bld/openmm_1482212188316/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/bin/cmake
  -E cmake_link_script CMakeFiles/cmTC_5ebcb.dir/link.txt --verbose=1
  /opt/clang/bin/clang -g -O3 -I/hbb_shlib/include -L/hbb_shlib/lib
  -static-libstdc++ CMakeFiles/cmTC_5ebcb.dir/testCCompiler.c.o -o cmTC_5ebcb
  -rdynamic
  clang-3.8: warning: argument unused during compilation: '-static-libstdc++'
  /opt/rh/devtoolset-2/root/usr/bin/ld: cannot find crtbegin.o: No such file
  or directory
  /opt/rh/devtoolset-2/root/usr/bin/ld: cannot find -lgcc
  /opt/rh/devtoolset-2/root/usr/bin/ld: cannot find -lgcc_s
  clang-3.8: error: linker command failed with exit code 1 (use -v to see
  invocation)
  gmake[1]: *** [cmTC_5ebcb] Error 1
  gmake[1]: Leaving directory
  `/anaconda/conda-bld/openmm_1482212188316/work/build/CMakeFiles/CMakeTmp'
  gmake: *** [cmTC_5ebcb/fast] Error 2
  
  
  CMake will not be able to correctly generate this project.
Call Stack (most recent call first):
  CMakeLists.txt:21 (PROJECT)
```

**Comment by jchodera** [comment]
I'll try to build locally and push, but would like to figure out why this isn't working on travis.

**Comment by jchodera** [comment]
Now I'm super confused---everything built fine locally up through the Python packages when I tried this last night, but now I can't get past the `Check for working C compiler: /opt/clang/bin/clang -- broken` issue.

**Comment by peastman** [comment]
It's because the computer is out to get you.  Or as I've observed often before, Nothing Ever Works.

**Comment by mpharrigan** [comment]
did you make the symlinks to rename centos -> redhat (per other thread)

File: openmm/build.sh [hunk 1]
```diff
@@ -1,5 +1,46 @@
 #!/bin/bash
 
+# Fix hbb issues
+if [ ! -e /opt/rh/devtoolset-2/root/usr/lib/gcc/x86_64-CentOS-linux/ ]; then
```
Comment (mpharrigan) [inline openmm/build.sh]: this isn't the condition you want to check. Do you need a check like this at all? If yes, remove the if statement and change the next line to `ln -sf` but I can't see why that would be necessary if we start from a fresh image every time

**Comment by jchodera** [comment]
Maybe that is the problem. I was debugging inside an image and didn't want to dump it each time, but I'll take that out.

**Comment by jchodera** [comment]
Thanks, @mpharrigan! Got a bit further, and now it tries to use Anaconda python, but still segfaults with an `invalid pointer` error.
```
+ /anaconda/conda-bld/openmm_1482272335619/_t_env/bin/python -s /anaconda/conda-bld/openmm_1482272335619/test_tmp/run_test.py
===== testing package: openmm-7.1.0-py27_0 =====
import: 'simtk'
import: 'simtk.openmm'
*** glibc detected *** /anaconda/conda-bld/openmm_1482272335619/_t_env/bin/python: free(): invalid pointer: 0x00007f41d0b70da0 ***
======= Backtrace: =========
/lib64/libc.so.6[0x7f41d420d4af]
/lib64/libc.so.6(cfree+0x4b)[0x7f41d42117ab]
/anaconda/conda-bld/openmm_1482272335619/_t_env/lib/plugins/libOpenMMCUDA.so[0x7f41cfdd0539]
/anaconda/conda-bld/openmm_1482272335619/_t_env/lib/plugins/libOpenMMCUDA.so(_ZN6OpenMM12CudaPlatformC1Ev+0x13e)[0x7f41cfd8588e]
/anaconda/conda-bld/openmm_1482272335619/_t_env/lib/plugins/libOpenMMCUDA.so(registerPlatforms+0x19)[0x7f41cfd85729]
/anaconda/conda-bld/openmm_1482272335619/_t_env/lib/python2.7/site-packages/simtk/openmm/../../../../libOpenMM.so(_ZN6OpenMM8Platform24loadPluginsFromDirectoryERKSs+0x5f4)[0x7f41d1dd9344]
/anaconda/conda-bld/openmm_1482272335619/_t_env/lib/python2.7/site-packages/simtk/openmm/_openmm.so[0x7f41d234013c]
/anaconda/conda-bld/openmm_1482272335619/_t_env/bin/../lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x7b35)[0x7f41d4e981e5]
/anaconda/conda-bld/openmm_1482272335619/_t_env/bin/../lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x8c95)[0x7f41d4e99345]
/anaconda/conda-bld/openmm_1482272335619/_t_env/bin/../lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x89e)[0x7f41d4e99c3e]
/anaconda/conda-bld/openmm_1482272335619/_t_env/bin/../lib/libpython2.7.so.1.0(PyEval_EvalCode+0x32)[0x7f41d4e99d52]
/anaconda/conda-bld/openmm_1482272335619/_t_env/bin/../lib/libpython2.7.so.1.0(PyImport_ExecCodeModuleEx+0xc2)[0x7f41d4eabbd2]
/anaconda/conda-bld/openmm_1482272335619/_t_env/bin/../lib/libpython2.7.so.1.0[0x7f41d4eae31e]
/anaconda/conda-bld/openmm_1482272335619/_t_env/bin/../lib/libpython2.7.so.1.0[0x7f41d4eaeb7a]
/anaconda/conda-bld/openmm_1482272335619/_t_env/bin/../lib/libpython2.7.so.1.0[0x7f41d4eaf0d1]
/anaconda/conda-bld/openmm_1482272335619/_t_env/bin/../lib/libpython2.7.so.1.0[0x7f41d4eaf344]
/anaconda/conda-bld/openmm_1482272335619/_t_env/bin/../lib/libpython2.7.so.1.0(PyImport_ImportModuleLevel+0x248)[0x7f41d4eaf9d8]
/anaconda/conda-bld/openmm_1482272335619/_t_env/bin/../lib/libpython2.7.so.1.0[0x7f41d4e8f2cf]
/anaconda/conda-bld/openmm_1482272335619/_t_env/bin/../lib/libpython2.7.so.1.0(PyObject_Call+0x53)[0x7f41d4de4e93]
/anaconda/conda-bld/openmm_1482272335619/_t_env/bin/../lib/libpython2.7.so.1.0(PyEval_CallObjectWithKeywords+0x43)[0x7f41d4e8f7b3]
/anaconda/conda-bld/openmm_1482272335619/_t_env/bin/../lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x3d96)[0x7f41d4e94446]
/anaconda/conda-bld/openmm_1482272335619/_t_env/bin/../lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x89e)[0x7f41d4e99c3e]
/anaconda/conda-bld/openmm_1482272335619/_t_env/bin/../lib/libpython2.7.so.1.0(PyEval_EvalCode+0x32)[0x7f41d4e99d52]
/anaconda/conda-bld/openmm_1482272335619/_t_env/bin/../lib/libpython2.7.so.1.0(PyRun_FileExFlags+0xb0)[0x7f41d4eba450]
/anaconda/conda-bld/openmm_1482272335619/_t_env/bin/../lib/libpython2.7.so.1.0(PyRun_SimpleFileExFlags+0xef)[0x7f41d4eba62f]
/anaconda/conda-bld/openmm_1482272335619/_t_env/bin/../lib/libpython2.7.so.1.0(Py_Main+0xca4)[0x7f41d4ecffd4]
/lib64/libc.so.6(__libc_start_main+0xf4)[0x7f41d41b99f4]
/anaconda/conda-bld/openmm_1482272335619/_t_env/bin/python[0x400729]
```

**Comment by jchodera** [comment]
Maybe this is an issue of the wrong `swig` being used?

**Comment by jchodera** [comment]
@mpharrigan : Did you ever get it to proceed through the python build without that segfault?

**Comment by jchodera** [comment]
Hm. [This thread](https://github.com/ContinuumIO/topik/issues/71#issuecomment-185392892) suggests that anaconda python is build for GCC 4, and that GCC 5.2 is binary-incompatible with GCC 4. 

@peastman : Any ideas?

**Comment by peastman** [comment]
No idea.  I thought you already had this working.

I'm not quite clear what the issue is.  You're using clang, not gcc, right?  What version?  And what version of clang introduced the new ABI?  Also, doesn't it have an option to tell it which ABI to compile for?

**Comment by jchodera** [comment]
> No idea. I thought you already had this working.

@mpharrigan had reported that he got it to work manually, and I just had to put his changes into the automated build system (or so I thought) by reconstructing the hints he gave in [this thread](https://github.com/pandegroup/openmm/issues/1613).

> I'm not quite clear what the issue is. 

Compilation of OpenMM succeeds, but building the Python wrappers with swig results in a segfault, with the stack trace as above.

> You're using clang, not gcc, right? 

Yes, `clang/clang++` is used for the build. You can see that [here](https://travis-ci.org/omnia-md/conda-dev-recipes/jobs/185627113#L585-L596).

> What version? 

We install [clang 3.8.1](https://github.com/omnia-md/omnia-build-box/blob/cuda80-amd30-clang38/Dockerfile#L6-L23) into the docker image containing CUDA 8.0 and AMD APP SDK 3.0.

> And what version of clang introduced the new ABI? 
> Also, doesn't it have an option to tell it which ABI to compile for?

I think this problem exists with GCC 5:
* http://allanmcrae.com/2015/06/the-case-of-gcc-5-1-and-the-two-c-abis/
* https://llvm.org/bugs/show_bug.cgi?id=23529

If anaconda is still being built with GCC 4, however, that shouldn't be an issue, should it?

**Comment by mpharrigan** [comment]
> I think this problem exists with GCC 5:

I think @peastman is confused because gcc 5 is being used nowhere in our system

**Comment by mpharrigan** [comment]
Yes, I got it to build, but I never tried importing the python package, which seems to be what triggers the error here, right?

**Comment by peastman** [comment]
What I mean is, you're building OpenMM with clang, and that needs to be compatible with anaconda python, which was built with gcc 4.  As long as clang is using the old (gcc 4) ABI, it should be fine.  What version of clang started supporting the new (gcc 5) ABI?  Older versions of clang will automatically build for the old ABI, so that should be fine.  And I think newer versions of clang let you specify which one to build for.

**Comment by jchodera** [comment]
> Yes, I got it to build, but I never tried importing the python package, which seems to be what triggers the error here, right?

You're right---it's failing in the testing stage, not the run stage. It could be a weird issue with trying to run CUDA in the docker container on travis. Let me try disabling the tests to push the package and see what happens when we try to use it locally.

> What I mean is, you're building OpenMM with clang, and that needs to be compatible with anaconda python, which was built with gcc 4. As long as clang is using the old (gcc 4) ABI, it should be fine. What version of clang started supporting the new (gcc 5) ABI? Older versions of clang will automatically build for the old ABI, so that should be fine. And I think newer versions of clang let you specify which one to build for.

I'm honestly not sure here. The [clang ticket about GCC 5 compatibility ABI support](https://llvm.org/bugs/show_bug.cgi?id=23529) is still open, it seems.

**Comment by jchodera** [comment]
Nope. This is what I get with the built package:
```
[chodera@gpu-2-10 ~]$ python
Python 3.5.2 |Continuum Analytics, Inc.| (default, Jul  2 2016, 17:53:06) 
[GCC 4.4.7 20120313 (Red Hat 4.4.7-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import simtk.openmm
*** glibc detected *** python: free(): invalid pointer: 0x00007f13c4e05da0 ***
======= Backtrace: =========
/lib64/libc.so.6[0x385a275f3e]
/lib64/libc.so.6[0x385a278d8d]
/cbio/jclab/home/chodera/miniconda3/lib/plugins/libOpenMMCUDA.so(+0x1c3539)[0x7f13c4065539]
/cbio/jclab/home/chodera/miniconda3/lib/plugins/libOpenMMCUDA.so(_ZN6OpenMM12CudaPlatformC1Ev+0x13e)[0x7f13c401a88e]
/cbio/jclab/home/chodera/miniconda3/lib/plugins/libOpenMMCUDA.so(registerPlatforms+0x19)[0x7f13c401a729]
/cbio/jclab/home/chodera/miniconda3/lib/python3.5/site-packages/simtk/openmm/../../../../libOpenMM.so(_ZN6OpenMM8Platform24loadPluginsFromDirectoryERKSs+0x5f4)[0x7f13cc62f344]
/cbio/jclab/home/chodera/miniconda3/lib/python3.5/site-packages/simtk/openmm/_openmm.cpython-35m-x86_64-linux-gnu.so(+0xf868c)[0x7f13ccb9668c]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyCFunction_Call+0xf9)[0x7f13cdae65e9]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyEval_EvalFrameEx+0x8fb5)[0x7f13cdb6dbd5]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyEval_EvalFrameEx+0x9546)[0x7f13cdb6e166]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(+0x144b49)[0x7f13cdb6eb49]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyEval_EvalCodeEx+0x48)[0x7f13cdb6ecd8]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyEval_EvalCode+0x3b)[0x7f13cdb6ed1b]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(+0x137dfe)[0x7f13cdb61dfe]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyCFunction_Call+0xf9)[0x7f13cdae65e9]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyEval_EvalFrameEx+0x6ba0)[0x7f13cdb6b7c0]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(+0x144b49)[0x7f13cdb6eb49]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyEval_EvalFrameEx+0x91d5)[0x7f13cdb6ddf5]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyEval_EvalFrameEx+0x9546)[0x7f13cdb6e166]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyEval_EvalFrameEx+0x9546)[0x7f13cdb6e166]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyEval_EvalFrameEx+0x9546)[0x7f13cdb6e166]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(+0x144b49)[0x7f13cdb6eb49]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyEval_EvalCodeEx+0x48)[0x7f13cdb6ecd8]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(+0x9a542)[0x7f13cdac4542]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyObject_Call+0x56)[0x7f13cda91236]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(_PyObject_CallMethodIdObjArgs+0xf8)[0x7f13cda94098]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyImport_ImportModuleLevelObject+0x403)[0x7f13cdb85ef3]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(+0x13872f)[0x7f13cdb6272f]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyCFunction_Call+0x131)[0x7f13cdae6621]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyObject_Call+0x56)[0x7f13cda91236]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyEval_CallObjectWithKeywords+0x43)[0x7f13cdb631b3]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyEval_EvalFrameEx+0x4f88)[0x7f13cdb69ba8]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(+0x144b49)[0x7f13cdb6eb49]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyEval_EvalCodeEx+0x48)[0x7f13cdb6ecd8]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyEval_EvalCode+0x3b)[0x7f13cdb6ed1b]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyRun_InteractiveOneObject+0x1e6)[0x7f13cdb96276]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyRun_InteractiveLoopFlags+0x5e)[0x7f13cdb965de]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(PyRun_AnyFileExFlags+0x4c)[0x7f13cdb9671c]
/cbio/jclab/home/chodera/miniconda3/bin/../lib/libpython3.5m.so.1.0(Py_Main+0xde2)[0x7f13cdbb0a02]
python(main+0x15d)[0x400add]
/lib64/libc.so.6(__libc_start_main+0xfd)[0x385a21ed1d]
python[0x4008b9]
======= Memory map: ========
00400000-00401000 r-xp 00000000 00:1c 9000582                            /cbio/jclab/home/chodera/miniconda3/bin/python3.5
00601000-00602000 rw-p 00001000 00:1c 9000582                            /cbio/jclab/home/chodera/miniconda3/bin/python3.5
01604000-01fc2000 rw-p 00000000 00:00 0                                  [heap]
3859e00000-3859e20000 r-xp 00000000 09:00 1048629                        /lib64/ld-2.12.so
385a01f000-385a020000 r--p 0001f000 09:00 1048629                        /lib64/ld-2.12.so
385a020000-385a021000 rw-p 00020000 09:00 1048629                        /lib64/ld-2.12.so
385a021000-385a022000 rw-p 00000000 00:00 0 
385a200000-385a38a000 r-xp 00000000 09:00 1048656                        /lib64/libc-2.12.so
385a38a000-385a58a000 ---p 0018a000 09:00 1048656                        /lib64/libc-2.12.so
385a58a000-385a58e000 r--p 0018a000 09:00 1048656                        /lib64/libc-2.12.so
385a58e000-385a590000 rw-p 0018e000 09:00 1048656                        /lib64/libc-2.12.so
385a590000-385a594000 rw-p 00000000 00:00 0 
385a600000-385a683000 r-xp 00000000 09:00 1048789                        /lib64/libm-2.12.so
385a683000-385a882000 ---p 00083000 09:00 1048789                        /lib64/libm-2.12.so
385a882000-385a883000 r--p 00082000 09:00 1048789                        /lib64/libm-2.12.so
385a883000-385a884000 rw-p 00083000 09:00 1048789                        /lib64/libm-2.12.so
385aa00000-385aa02000 r-xp 00000000 09:00 1048803                        /lib64/libdl-2.12.so
385aa02000-385ac02000 ---p 00002000 09:00 1048803                        /lib64/libdl-2.12.so
385ac02000-385ac03000 r--p 00002000 09:00 1048803                        /lib64/libdl-2.12.so
385ac03000-385ac04000 rw-p 00003000 09:00 1048803                        /lib64/libdl-2.12.so
385ae00000-385ae17000 r-xp 00000000 09:00 1048783                        /lib64/libpthread-2.12.so
385ae17000-385b017000 ---p 00017000 09:00 1048783                        /lib64/libpthread-2.12.so
385b017000-385b018000 r--p 00017000 09:00 1048783                        /lib64/libpthread-2.12.so
385b018000-385b019000 rw-p 00018000 09:00 1048783                        /lib64/libpthread-2.12.so
385b019000-385b01d000 rw-p 00000000 00:00 0 
385b600000-385b607000 r-xp 00000000 09:00 1048784                        /lib64/librt-2.12.so
385b607000-385b806000 ---p 00007000 09:00 1048784                        /lib64/librt-2.12.so
385b806000-385b807000 r--p 00006000 09:00 1048784                        /lib64/librt-2.12.so
385b807000-385b808000 rw-p 00007000 09:00 1048784                        /lib64/librt-2.12.so
385c200000-385c22e000 r-xp 00000000 09:00 1048695                        /lib64/libncursesw.so.5.7
385c22e000-385c42d000 ---p 0002e000 09:00 1048695                        /lib64/libncursesw.so.5.7
385c42d000-385c42e000 rw-p 0002d000 09:00 1048695                        /lib64/libncursesw.so.5.7
385ee00000-385ee1d000 r-xp 00000000 09:00 1048662                        /lib64/libtinfo.so.5.7
385ee1d000-385f01c000 ---p 0001d000 09:00 1048662                        /lib64/libtinfo.so.5.7
385f01c000-385f020000 rw-p 0001c000 09:00 1048662                        /lib64/libtinfo.so.5.7
385f020000-385f021000 rw-p 00000000 00:00 0 
385fe00000-385fe02000 r-xp 00000000 09:00 1048854                        /lib64/libutil-2.12.so
385fe02000-3860001000 ---p 00002000 09:00 1048854                        /lib64/libutil-2.12.so
3860001000-3860002000 r--p 00001000 09:00 1048854                        /lib64/libutil-2.12.so
3860002000-3860003000 rw-p 00002000 09:00 1048854                        /lib64/libutil-2.12.so
7f13b0000000-7f13b0021000 rw-p 00000000 00:00 0 
7f13b0021000-7f13b4000000 ---p 00000000 00:00 0 
7f13b6e56000-7f13b6f38000 r-xp 00000000 00:1c 200873562                  /cbio/jclab/home/chodera/miniconda3/lib/plugins/libOpenMMAmoebaReference.so
7f13b6f38000-7f13b7137000 ---p 000e2000 00:1c 200873562                  /cbio/jclab/home/chodera/miniconda3/lib/plugins/libOpenMMAmoebaReference.so
7f13b7137000-7f13b713e000 rw-p 000e1000 00:1c 200873562                  /cbio/jclab/home/chodera/miniconda3/lib/plugins/libOpenMMAmoebaReference.so
7f13b713e000-7f13b7140000 rw-p 00000000 00:00 0 
7f13b7140000-7f13b714d000 rw-p 004e9000 00:1c 200873562                  /cbio/jclab/home/chodera/miniconda3/lib/plugins/libOpenMMAmoebaReference.so
7f13b714d000-7f13b7161000 r-xp 00000000 00:1c 200873545                  /cbio/jclab/home/chodera/miniconda3/lib/plugins/libOpenMMDrudeReference.so
7f13b7161000-7f13b7361000 ---p 00014000 00:1c 200873545                  /cbio/jclab/home/chodera/miniconda3/lib/plugins/libOpenMMDrudeReference.so
7f13b7361000-7f13b7362000 rw-p 00014000 00:1c 200873545                  /cbio/jclab/home/chodera/miniconda3/lib/plugins/libOpenMMDrudeReference.so
7f13b7362000-7f13b7364000 rw-p 00091000 00:1c 200873545                  /cbio/jclab/home/chodera/miniconda3/lib/plugins/libOpenMMDrudeReference.so
7f13b7364000-7f13b7377000 r-xp 00000000 00:1c 200873542                  /cbio/jclab/home/chodera/miniconda3/lib/plugins/libOpenMMRPMDReference.sAbort
```

**Pull request merged**